packages/dev/core/src/Behaviors/Meshes/pointerDragBehavior.ts

```diff
@@ -452,7 +452,7 @@ export class PointerDragBehavior implements Behavior<AbstractMesh> {
 
                 // Project delta drag from the drag plane onto the drag axis
                 pickedPoint.subtractToRef(this.lastDragPosition, this._tmpVector);
-                dragLength = Vector3.Dot(this._tmpVector, this._worldDragAxis);
+                dragLength = Vector3.Dot(this._tmpVector, this._worldDragAxis) / (Vector3.Dot(this._worldDragAxis, this._worldDragAxis) || 1);

```

Comment:
Thanks for the bugfix!  Instead of dividing the dot product by the length of this._worldDragAxis squared, could you please normalize this._worldDragAxis before this line?  Something like:

Vector3.NormalizeToRef(this._worldDragAxis, this._worldDragAxis);

Then this line can remain unchanged:
dragLength = Vector3.Dot(this._tmpVector, this._worldDragAxis);

Thanks again!